deploy/deployment-manager/service_account.py

```diff
@@ -0,0 +1,65 @@
+"""A template file for deploying a service account"""
+
+
+def generate_config(context):
+    """Generates service account user"""
+    deployment_name = context.env["deployment"]
+    scope = context.properties["scope"]
+    parent_id = context.properties["parentId"]
+    sa_name = context.properties["serviceAccountName"]
+    roles = ["roles/cloudasset.viewer", "roles/browser"]
+
+    # pylint: disable=duplicate-code
+    service_account = {
```

Comment:
it'd be nice to define a `./common.py` file and import it to both deployment templates. not sure why that didn't work tbh

Reply:
I didn't manage to do it as well.
It's because we're using deployment manager templates and other py files can not be imported to the file.